addons/test_mass_mailing/tests/test_mailing.py

```diff
@@ -28,7 +28,7 @@ def test_mailing_gateway_reply(self):
         mailing = self.env['mailing.mailing'].create({
             'name': 'TestName',
             'subject': 'TestSubject',
-            'body_html': 'Hello ${object.name}',
+            'body_html': 'Hello {{ object.name }}',
```

Comment:
Wrong change as body should contain qweb, not inline template (to propagate to other tests, re-check a bit all of them to be sure they makes sense :) )